src/pages/home/sidebar/SidebarScreen.js

```diff
@@ -170,5 +180,8 @@ export default compose(
         betas: {
             key: ONYXKEYS.BETAS,
         },
+        isNewUser: {
+            key: ONYXKEYS.IS_NEW_USER
+        }
```

Comment:
We like trailing commas :D

```suggestion
        isNewUser: {
            key: ONYXKEYS.IS_NEW_USER,
        },
```